### PR TITLE
[SurojSultonov] - Likes for blog posts

### PR DESCRIPTION
**Pull request by SurajSulten**

### CODE REVIEW

File: src/blog/blogLikes.entity.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
```
Comment (mukhammadjon-ss) [inline src/blog/blogLikes.entity.ts:1]: I think you don't need to create separate entities for likes and dislikes. dislike works only when user already liked post. It is used to return his/her like

File: src/blog/blog.controller.ts [hunk 2]
```diff
@@ -40,6 +42,21 @@ export class BlogController {
...
+    static async getBlogById(req: Request, res: Response) {
+        const {id} = req.params
+
+        const blogRepository = AppDataSource.getRepository(Blog);
+        const blog = await blogRepository.findOne({where: {id}})
+
+        if(!blog) {
+            return res.status(404).json({message: "Blog not found"})
+        }
+        console.log(blog);
```
Comment (mukhammadjon-ss) [inline src/blog/blog.controller.ts]: please remove console.log

File: src/blog/blog.controller.ts [hunk 3]
```diff
@@ -95,8 +112,78 @@ export class BlogController {
...
         if (!blog) return res.status(404).json({ message: 'Blog not found' });
 
-        await blogRepository.remove(blog); // Remove the blog
+        await blogRepository.remove(blog); 
 
         return res.status(200).json({ message: 'Blog deleted successfully' });
     }
-}
+
+    static async likeBlog(req: PrivateRequest, res: Response) {
```
Comment (mukhammadjon-ss) [inline src/blog/blog.controller.ts:119]: please save record only when user likes the blog. This record is used to identify which user liked particular post. when user clicks 'thumbs up' button, you will check whether user already like this post. If he didn't like yet (no record of user and post), then you insert one, if this record already exists, you delete this record (which means he decided to unlike post)

**Review by mukhammadjon-ss — CHANGES_REQUESTED** [review]

File: src/blog/blogLikes.entity.ts (hunk 1 above)
Comment (SurajSulten) [reply src/blog/blogLikes.entity.ts:1]: removed dislike

File: src/blog/blog.controller.ts (hunk 2 above)
Comment (SurajSulten) [reply src/blog/blog.controller.ts]: removed

File: src/blog/blog.controller.ts (hunk 3 above)
Comment (SurajSulten) [reply src/blog/blog.controller.ts:119]: removed dislike

**Pull request merged**